multinode/transaction_sender.go

```diff
@@ -23,25 +23,26 @@ var (
 	}, []string{"network", "chainId", "invariant"})
 )
 
-type SendTxResult interface {
-	Code() SendTxReturnCode
-	Error() error
+type sendTxResult[RESULT any] struct {
```

Comment:
Instead of requiring this interface to be implemented by each user, we can use a struct internally, and multivalue returns with a plain generic result value. This shed a ton of boilerplate, including the constructor function.